lib/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const UBUNTU_FLEET_ARCHS: string[] = ['x86_64_ubuntu', 'arm64_ubuntu'];
+
+// curl -s https://api.github.com/users/<username> | jq '.id'
+export const GITHUB_ALLOWLISTED_ACCOUNT_IDS = [
```

Comment:
Let's add a note that these accounts are members of the runfinch org (+dependabot). Maybe a TODO comment to automate fetching these account ids from the org?